reference/resources/browser_info.yaml

```
@@ -0,0 +1,67 @@
+---
+title: Browser info
+type: object
+
+required:
+  - java_enabled
+  - javascript_enabled
+  - language
+  - color_depth
+  - screen_height
+  - screen_width
+  - time_zone_offset
+  - user_device
+  - user_agent
+
+properties:
+  java_enabled:
+    type: boolean
+    description: Indicates whether the client browser supports Java.
+    example: true
+
+  javascript_enabled:
+    type: boolean
+    description: Indicates whether the client browser supports JavaScript.
+    example: true
+
+  language:
+    type: string
+    description: |-
+      The preferred language of the buyer,
+      usually the language of the browser UI.
+    example: "en-GB"
+
+  color_depth:
+    type: number
+    description: The color depth of the screen.
+    example: 32
+
+  screen_height:
+    type: number
+    description: The height of the screen in pixels.
+    example: 1080
+
+  screen_width:
+    type: number
+    description: The width of the screen in pixels.
+    example: 1920
+
+  time_zone_offset:
+    type: number
+    description: Time-zone offset in minutes between UTC and buyer location.
+    example: 60
+
+  user_device:
+    type: string
+    description: The platform that is being used to access the website.
+    example: "desktop"
+    enum:
+      - "desktop"
+      - "mobile"
+
+  user_agent:
+    type: string
+    description: The user agent string for the current browser.
+    example: |-
+      Mozilla/5.0 (darwin) AppleWebKit/537.36
+      (KHTML, like Gecko) jsdom/16.7.0
```

```suggestion
      (KHTML, like Gecko) jsdom/16.7.0
  accept_header:
    type: string
    description: The `Accept` header of the request from the buyer's browser.
    example: "*/*"
```
Sorry @tculigGr4vy we also need to add an optional `accept_header` property following a change to TA-1176. We won't pass this a value from Embed, in which case it will default to using the request header.